.github/workflows/ansible-ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
```

```suggestion
    steps:
    - uses: actions/checkout@v2
      with:
        repository:  https://github.com/adfinis-sygroup/ansible-lint-rules
        path: ../lint-rules
```

@tongpu According to the [documentation](https://github.com/actions/checkout)

---

Let's try it.